utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -164,30 +163,49 @@ object SwaggerTyped {
     case SwaggerNull =>
       TypedNull
   }
+
+  private def handleSwaggerObject(elementType: Map[PropertyName, SwaggerTyped], additionalProperties: AdditionalProperties, patternProperties: Map[Pattern, SwaggerTyped]): TypingResult = {
+    import pl.touk.nussknacker.engine.util.Implicits.RichScalaMap
+    def typedStringKeyMap(valueType: TypingResult) = {
+      Typed.genericTypeClass(classOf[util.Map[_, _]], List(Typed[PropertyName], valueType))
+    }
+    if (elementType.isEmpty) {
+      val patternPropertiesTypesSet = patternProperties.values.map(typingResult).toSet
+      additionalProperties match {
+        case AdditionalPropertiesDisabled if patternPropertiesTypesSet.isEmpty => TypedObjectTypingResult(List.empty)
+        case AdditionalPropertiesDisabled => typedStringKeyMap(Typed(patternPropertiesTypesSet))
+        case AdditionalPropertiesWithoutType => typedStringKeyMap(Unknown)
```

Comment:
here, you dont want to handle patternProperties?

Reply:
nope, but thanks for ensuring